ee/clickhouse/queries/clickhouse_session_recording.py

```diff
@@ -33,9 +33,11 @@
 
 
 class SessionRecording(BaseSessionRecording):
-    def query_recording_snapshots(self, team: Team, session_id: str) -> List[Any]:
+    def query_recording_snapshots(self, team: Team, session_id: str) -> Tuple[Optional[str], List[Any]]:
```

Comment:
Very minor thing, maybe worth adding a small comment clarifying that the returned tuple will contain the session's ID & the snapshot data?

Reply:
Did one better - named the types :)